packages/dotnet/ReadMe/HarJsonTranslationLogics/HarJsonBuilder.cs

```diff
@@ -101,35 +101,20 @@ private async Task<Response> BuildResponse()
     private Creator BuildCreator()
     {
       Creator creator = new Creator();
-      creator.name = ConstValues.name;
+      creator.name = "readme-metrics (dotnet)";
       creator.version = ConstValues.version;
-      creator.comment = GetOS();
+      creator.comment = GetCreatorVersion();
       return creator;
     }
-    private string GetOS()
+
+    /**
+     * @example x86-win32nt/6.2.9200.0
+     */
+    private string GetCreatorVersion()
     {
-      string os = null;
-      if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-      {
-        os = "windows";
-      }
-      else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
-      {
-        os = "mac";
-      }
-      else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
-      {
-        os = "linux";
-      }
-      else
-      {
-        os = "unknown";
-      }
-      os = os + "/" + Environment.OSVersion.Version;
-      return os;
+      return RuntimeInformation.OSArchitecture + "/" + Environment.OSVersion.Platform + "/" + Environment.OSVersion.Version;
```

Comment:
I've updated our other SDKs to have the `require('os').release()` equivalent in here after the OS platform but my googlefu is failing to find any alternative to this in .NET.

Reply:
I was also unable to run the Docker integration suite locally for this to ensure that the string I'm generating here looks like the `x86-win32nt/6.2.9200.0` example I think I'm generating here.

![Screen Shot 2022-08-15 at 12 29 37 PM](https://user-images.githubusercontent.com/33762/184703917-be9f1908-93f4-4fe0-92f5-947575769b5e.png)